### DISCUSSION TITLE
Add is_favoriteproducts without compiling theme

### DISCUSSION THREAD

**Discussion by Shoprunners** [Oksydan/falcon#352] [General]
Hi,
sorry, I am not an expert in webpack and I am not sure how to achieve this:
I created a theme based on falcon did lots of customization and customer wants to use the is_favoriteproducts module.
As far as I see, the css and js ressources are not compiled/webpacked in the release. 
Is there a way to add this module to the shop, without compiling the whole theme again? Maybe just use webpack on the new module and ignore the rest? any hints are welcome.

**Comment by Oksydan** [comment]
Hi @Shoprunners,

compiling it via webpack is the only way.  We are using theme scss variables and in next version even theme js components inside module source files. It's the only way and It will be the only way. Compiling theme isn't hard. All you have to do is to just install dependencies via npm and run build script. Give it a try, w/o using webpack it is meaningless to use Falcon.

**Reply by Shoprunners** [reply]
okay, thanks for clarification. I manged to set it up like you suggested.
